site/content/en/docs/Guides/Client SDKs/unity.md

```diff
@@ -215,7 +194,6 @@ This returns a list of the playerIDs that are currently connected to the GameSer
 List<string> players = await agones.GetConnectedPlayers();
 ```
 
-{{% /feature %}}
```

Comment:
This seems very odd, as you deleted the end of a feature code, but nothing from the beginning?

I figure this has broken lots of stuff in your PR. (Same in all the other ones as well)